invenio_users_resources/services/results.py

```diff
@@ -58,5 +58,5 @@ def last_modified(self):
 
     @property
     def max_age(self):
-        """Get last modified date for the response."""
-        return 86000 * 7
+        """Get max age for cached resource in seconds."""
+        return 60 * 5
```

Comment:
Why did you go with 60 * 5?

Reply:
Agree, [magic numbers do not help](https://en.wikipedia.org/wiki/Magic_number_(programming)#Unnamed_numerical_constants)... can we give it a name and a short explanation why if the name is not self-explanatory?